File: template/Python/fizz-buzz.py
#!/usr/bin/env python

for i in range(1, FIZZBUZZ_CYCLES+1):
    def divides(mod):
        return i % mod == 0
    div3 = divides(3)
    div5 = divides(5)

    if div3:
        if div5:
            print("FizzBuzz")
        else:
            print("Fizz")
    elif div5:
        print("Buzz")
    else:
        print(i)

File: template/JavaScript/fizz-buzz.js
for (var i = 1; i <= FIZZBUZZ_CYCLES; ++i) {
	const div3 = i % 3 == 0;
	const div5 = i % 5 == 0;
	console.log(div3 && div5 ? "FizzBuzz" : div3 ? "Fizz" : div5 ? "Buzz" : i);
}
